### ISSUE TITLE
I dont quit get

### ISSUE THREAD

**Issue by eshe-huli** [angelabauer/Story#24]
Can you explain what this repo is about?

**Comment by fbitti** [comment]
Just an educated guess, it seems to be a repo where Angela's students learned about pull requests on GitHub and practiced.

**Comment by vsasvipul0605** [comment]
The repo where she helps her _students_ learning about initials of **git** and **github**.

**Comment by thefatmenon** [comment]
Yup... That's what it is... May be I should make it private or something.

**Comment by kanyshaiosmonova** [comment]
> Just an educated guess, it seems to be a repo where Angela's students learned about pull requests on GitHub and practiced.

Yes, that's right!
